library/gcm.c

```diff
@@ -546,6 +547,7 @@ int mbedtls_gcm_finish( mbedtls_gcm_context *ctx,
      * for the sake of alternative implementations. */
     (void) output;
     (void) output_size;
+    *output_length = 0;
```

Comment:
Would it be worth repeating the comment here that our implentation does not support delaying up to a block of data, thus this will remain zero (but this was added for the sake of alt implementations, etc)

Reply:
Er, it says so in the comment just above?

Reply:
Oh. Github wasn't wanting to show me this.